package.json

```diff
@@ -5,5 +5,8 @@
     "grunt": "~0.4.4",
     "grunt-purescript": "~0.4.0",
     "grunt-contrib-clean": "~0.5.0"
+  },
+  "dependencies": {
+    "pulp": "^12.4.2"
```

Comment:
I'm probably going to move this to `devDependencies`. Similar to the `bower.json` comment, you don't have to do anything about it if you don't want to. Mentioning here so you're aware beforehand.